www/entrypoint.sh

```diff
@@ -1,4 +1,22 @@
-#!/bin/bash
+#!/bin/bash -e
+
+# generate certs
+mkdir -p /autocert/
+./autocert.sh /autocert/
+
+# link certs
+if [ ! -f /etc/ssl/private/desec.${DESECSTACK_DOMAIN}.cer ] ||
+   [ ! -f /etc/ssl/private/desec.${DESECSTACK_DOMAIN}.key ] ; then
```

Comment:
What if `desec.${DESECSTACK_DOMAIN}.{cer,key}` are there, but not one of the other certificates? (Why do these two files have a special role?)

More generally, I wondered why we are doing the `$CERT_PATH` magic instead of just making sure that `/etc/ssl/private/` has a complete set of key+certificate files? The simplest approach would be to have `autocert.sh` loop over the set of domain names, and generate any missing file pairs in `/etc/ssl/private/`.

Reply:
The `$CERT_PATH` magic is due to the fact that the certificate volume is mounted read-only (a feature that I wanted to keep).

Reply:
That is also the reason we can't mix and match user-provided and auto-generated certificates without an irresponsibly high effort.

Reply:
I kept the fact that we (due to read-only) cannot really mix auto-generated and user-provided certificates (for now?). I rewrote the condition in a way that all required files are checked, if a single one is missing, we fall back to autogeneration. The user is provided with a list of files that are missing in the certificate folder.